runtimes/CMakeLists.txt

```diff
@@ -9,6 +9,7 @@ include(${LLVM_COMMON_CMAKE_UTILS}/Modules/CMakePolicy.cmake
 include(${LLVM_COMMON_CMAKE_UTILS}/Modules/LLVMVersion.cmake)
 
 project(Runtimes C CXX ASM)
+set(LLVM_SUBPROJECT_TITLE "Runtimes")
```

Comment:
Is this needed? My understanding is that the runtimes CMake here is the root that includes all the other projects, so this doesn't actually contain any "projects'>

Reply:
Since LLVM is usually the top-level project everything is sorted under "LLVM" be default. I added a `LLVM_SUBPROJECT_TITLE` to all potential subprojects that are either included via `add_subdirectory` or as a stand-alone build, including this one.

I just checked to see if it is actually used, and the answer is yes: At least `check-runtimes` is put into this folder when loading the solution from the `runtimes-bins` directory.

I did not go through all runtime targets to ensure they are put into some folder name that makes sense like I did for the LLVM_ENABLE_PROJECTS subprojects since opening a project in the IDE from the runtimes-bin would be done only if there is a concrete problem. However, I wanted to at least make a minimal effort that allows puts targets other than `add_custom_target` into a subtproject folder.